### PR TITLE
Add admin sidebar

### PR DESCRIPTION
**Pull request by haunc-0912**
==> Tittle: Thêm sidebar cho admin layout

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/15880

==> Attachment File:
Screenshot from 2019-08-01 09-08-08

### CODE REVIEW

**Comment by haunc-0912** [comment]
ready

**Comment by huyhm-0966** [comment]
approved

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -141,31 +141,88 @@ footer {
...
+  }
+
+  &.navbar-default .navbar-nav > li > a {
+    color: $white;
+    &:hover {
+      color: $bl_3;
+    }
+  }
+}
+
```
Comment (huyhm-0966) [inline app/assets/stylesheets/custom.scss]: xóa bớt 1 line em

**Comment by huyhm-0966** [comment]
commented

**Comment by haunc-0912** [comment]
ready

**Pull request merged**